Extensions/retags.js

```diff
@@ -181,7 +217,12 @@ XKit.extensions.retags = {
 				});
 			}
 			$t.addClass(cls);
-			$c.append($retags);
+			if (!isPopover) {
+				$c.append($retags);
+				return;
+			}
+			$retags = "<li class='retags-wrapper'>" + $retags[0].outerHTML + "</li>";
```

Comment:
does this need to be an li? is there really no way we could just use CSS to make this work?

adding an "isPopover" parameter to this call feels like a code smell to me.